src/documents/tutorial-gettingStartedWithInfusion/DefineANamespaceAndCreateAClosure.md

```diff
@@ -15,11 +15,11 @@ Infusion code generally follows a few conventions that we recommend, and that we
 
 ### Namespacing ###
 
-We define a namespace for our code: a single global variable that becomes the container for the code. This helps keep code contained, meaning there's less chance of bad interactions with other code: Anything we want to be public will be attached to this object, so all of our code will be qualified by the namespace.
+We define a namespace for our code: a single global variable that becomes the container for the code. This means there's less chance of bad interactions with other code: Anything we want to be public will be attached to this object, so all of our code will be qualified by the namespace.
```

Comment:
Our general philosophy is to make everything public off the namespace. I guess the exception here is for variables within a function. Should we make a mention about this philosophy here?